src/assets/scss/pages/sweetalert2.scss

```diff
@@ -0,0 +1,3 @@
+.form-control {
```

Comment:
As you can see, based on [Netlify's deploy preview](https://deploy-preview-270--comforting-cocada-22086c.netlify.app/extra-component-sweetalert), the bug still occurs.
![image](https://github.com/zuramai/mazer/assets/76430369/006a116a-ef6f-4872-a7c6-2ea20d6a0b50)

That's because it affects CSS, so it has overridden it.
![image](https://github.com/zuramai/mazer/assets/76430369/a9da8836-b5c6-49d8-8b18-dedcd4d82c34)

Reply:
there's something strange bro. After I checked, why didn't the website load the SCSS that I had created?